applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -144,54 +144,78 @@ static void le_audio_event_publish(enum le_audio_evt_type event, struct bt_conn
  */
 static int headset_pres_delay_find(uint8_t index, uint32_t *pres_dly_us)
```

Comment:
Discuss testing.

Reply:
This is painful as we use Kconfigs to set all these value and our Kconfigs have great testing for validity of the pd and pref pd values. I have to comment out or hard code values that test the failure conditions. This code will probably all change when additions to the host are added.